src/stores/store.ts

```diff
@@ -272,9 +269,7 @@ export const useStore = defineStore('store', {
         this.setPlotMessage('dashboard.verifyingPlot');
         this.setNetworkMessage('dashboard.verifyingNet');
 
-        // TODO: remove variable assignment after backend (Rust) is updated
-        const isFarming = await client.startFarming(this.plotDir, this.plotSizeGB);
-        if (!isFarming) { throw Error; }
+        await client.startFarming(this.plotDir, this.plotSizeGB);
```

Comment:
nitpick: should we normalize `this.plotDir` as `this.plotPath`? We do this with `this.plotSizeGB`